lib/matplotlib/_mathtext.py

```diff
@@ -1236,11 +1236,11 @@ def __init__(self, elements):
         self.glue_order   = 0    # The order of infinity (0 - 3) for the glue
 
     def __repr__(self):
-        return '[%s <%.02f %.02f %.02f %.02f> %s]' % (
+        return '%s<w=%.02f h=%.02f d=%.02f s=%.02f>[%s]' % (
             super().__repr__(),
```

Comment:
```suggestion
        return '%r<w=%.02f h=%.02f d=%.02f s=%.02f>[%s]' % (
            super(),
```
is simpler.

Reply:
This doesn't work; the super object prints itself as `<super: <class 'List'>, <Hlist object>>`.